batchUploadResourceCollections.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This bash script is used to batch upload resource collections to the iHub server. It iterates through 
+
+# Assumes that you are in the root of the community directory and that it's named like /ihub-community-<COMMUNITY_NAME>
+community=$(basename "$(pwd)" | grep -o '[^-]*$') 
```

Comment:
The only repos that this would give unexpected results for are our internal repos (e.g. hub-community-testing-manual) so that's not a problem.

If the intent is to remove the `ihub-community` prefix an alternate way would be:

```
community=${$(basename $(pwd))##ihub-community-}
```

That's bash's way of stripping off a specific prefix.  I'm not requesting this change, just sharing another way.

Thanks for teaching me about grep's `-o` flag!

Reply:
Woah! That's pretty cool, I wasn't privy to `##` as a strip syntax. Thanks for the intel, and the caveat regarding the internal repos.